.testfaster.yml

```diff
@@ -47,7 +47,10 @@ base:
   os_dockerfile: |-
     FROM quay.io/testfaster/kube-ubuntu
     ENV cache-bust 19
-    RUN apt-get update && apt-get install -y build-essential gettext-base socat shellcheck golang-go rsync jq pv silversearcher-ag
+    RUN apt-get update && apt-get install -y build-essential gettext-base socat shellcheck rsync jq pv silversearcher-ag
+    RUN cd /root && wget -nv -O - https://dl.google.com/go/go1.15.4.linux-amd64.tar.gz | tar -zxf -
```

Comment:
this will fix some errors i've seen in circleci regarding some newer branches requiring 1.15, nice work!